public/js/loginValidation.js

```diff
@@ -0,0 +1,52 @@
+let isValidEmail = (elementID) => {
+    let inputVal = document.getElementById(elementID)
+    let userEmailWarning = document.getElementById('userEmailWarning')
+    let email = inputVal.value.trim();
+    let regEx = new RegExp("^\\w+([\\.-]?\\w+)*@\\w+([\\.-]?\\w+)*(\\.\\w{2,3})+$")
+    if (regEx.test(email)) {
+        userEmailWarning.classList.remove('error')
+        return true
+    } else {
+        userEmailWarning.classList.add('error')
+        return false
+    }
+}
+
+let validateInputs = () => {
```

Comment:
best practice is to pass in what you need as params

Reply:
encodeURI(document.getElementById('userEmail').value)
is being done twice - just make a function and use outside to generate params to pass in

Reply:
Function now has variables passed in.